MantidPlot/src/Mantid/MantidUI.cpp

```diff
@@ -97,6 +97,10 @@ size_t DET_TABLE_NDETS_GROUP = 10;
 
 // Initialize logger
 Mantid::Kernel::Logger g_log("MantidUI");
+
+bool isOfType(const QObject* obj, const char* toCompare) {
+  return strcmp(obj->metaObject()->className(), toCompare) == 0;
+}
```

Comment:
This looks like a very good idea.
Maybe it would be a good idea to revisit and harmonize these comparisons once all the groups are merged and the number of Qt3Support warnings is hopefully small/manageable.